pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -1401,10 +1403,12 @@ public bool ImportingSearch
             (c, p) => c.PeptideDigestMaxMissedCleavages = p.GetValueInt(DigestSettings.MIN_MISSED_CLEAVAGES, DigestSettings.MAX_MISSED_CLEAVAGES));
         public static readonly Argument ARG_PEPTIDE_UNIQUE_BY = DocArgument.FromEnumType<PeptideFilter.PeptideUniquenessConstraint>(@"pep-unique-by",
             (c, p) => c.PeptideDigestUniquenessConstraint = p);
-        public static readonly Argument ARG_BGPROTEOME_NAME = new DocArgument(@"background-proteome-name", () => GetDisplayNames(Settings.Default.BackgroundProteomeList),
-            (c, p) => c.BackgroundProteomeName = p.Value)
+        public static readonly Argument ARG_BGPROTEOME_NAME = new DocArgument(@"background-proteome-name",
+                () => Argument.ValuesToExample(Settings.Default.BackgroundProteomeList.Select(p => p.Name)
```

Comment:
This seems like a strange construct. Is it based on any other argument used to specify the name in an existing list?

Reply:
The isolation scheme argument also supports specifying a name or a file to import the scheme from. If you have a better idea for how to present that in the examples, let me know. I've considered putting it in angle brackets `<name to give protdb imported by --background_proteome_file>`.

Reply:
The isolation scheme argument is a single argument which makes it a bit simpler, but doesn't allow customizing the name of the isolation scheme if importing it from a raw file. Not sure which is preferable.